aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -340,6 +358,52 @@ void magmaTriangularSolveBatched<float>(
     const MAGMAQueue& magma_queue) {
   magmablas_strsm_batched(MagmaLeft, uplo, trans, diag, m, n, 1, dA_array, ldda, dB_array, lddb, batchsize, magma_queue.get_queue());
 }
+
+template<>
+inline magma_int_t magmaGeqrfOptimalBlocksize<double>(magma_int_t m, magma_int_t n) {
+  return magma_get_dgeqrf_nb(m, n);
+}
+
+template<>
+inline magma_int_t magmaGeqrfOptimalBlocksize<float>(magma_int_t m, magma_int_t n) {
+  return magma_get_sgeqrf_nb(m, n);
+}
+
+template<>
+void magmaGeqrf<double>(
+    magma_int_t m, magma_int_t n, double* dA, magma_int_t ldda,
+    double* tau, double* dT, magma_int_t* info, bool is_v2) {
+  if (!is_v2) {
```

Comment:
i think we can safely remove the `!is_v2` specializations. we stopped supporting magma v1 long ago

Reply:
The is_v2 specialization is to different between geqrf2 and geqrf. I will add a comment about this to avoid confusion.